aerogear/ionic-showcase
# fix: remove altair playground

## Motivation

Remove Altair as GraphQL playground now will contain extra stuff from Apollo Federation. We need that for experiments. Another reason is that Altair stopped allowing to execute multiple queries from the view.

## Verification steps

Please follow the verfication steps:
```
cd server
npm i
docker-compose up
npm run start 
```

Go to server url ( http://localhost:4000/graphql ) and you should see the old and still bad GraphQL playground :)


FYI @ankitjena @darahayes
